packages/apps/graph/src/graph/Query/hello.ts

```diff
@@ -0,0 +1,10 @@
+import { builder } from '../builder';
```

Comment:
Do we want to keep this in here as well?

Reply:
It's already removed, but didn't want to change this PR. Will be in next PR